src/domains/idpay-app/api/idpay_appio_full/openapi.appio.full.yml

```diff
@@ -1618,6 +1618,24 @@ paths:
               example:
                 code: PAYMENT_NOT_FOUND_OR_EXPIRED
                 message: transaction not found or expired
+        '412':
+          description: The transaction version mismatch
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TransactionErrorDTO'
+              example:
+                code: PAYMENT_TRANSACTION_VERSION_MISMATCH
+                message: The transaction version mismatch
+        '423':
```

Comment:
The same think for this kind of error. I think is more appropriate to return a REJECTED transaction or return 429, cause in this scenario the user can retry the payment until succedeed or expire due timeout

Reply:
ok, let's hide the 423 http status, we will return 429 and error CODE documented here